spaceship/lib/spaceship/client.rb

```diff
@@ -34,7 +34,31 @@ class InvalidUserCredentialsError < StandardError; end
     # Raised when no user credentials were passed at all
     class NoUserCredentialsError < StandardError; end
 
-    class UnexpectedResponse < StandardError; end
+    class UnexpectedResponse < StandardError
+      class << self
+        def handle_response!(response)
+          if response.body['userString']
+            require 'cgi'
+
+            result_string = CGI.unescapeHTML(response.body['resultString'])
+            user_string = CGI.unescapeHTML(response.body['userString'])
+            user_error!("Apple returned an unexpected response:\n\t#{result_string}\n\t#{user_string}")
```

Comment:
Maybe we should specify the actual host name? e.g. iTunes Connect or Apple Developer Portal